pkg/rootless/rootless_linux.go

```diff
@@ -110,6 +110,9 @@ func tryMappingTool(tool string, pid int, hostID int, mappings []idtools.IDMap)
 	args := []string{path, fmt.Sprintf("%d", pid)}
 	args = appendTriplet(args, 0, hostID, 1)
 	for _, i := range mappings {
+		if hostID >= i.HostID && hostID < i.HostID+i.Size {
+			return errors.Errorf("invalid configuration: the specified mapping %d:%d includes the host ID", i.HostID, i.Size)
```

Comment:
Does "host ID" mean UID? If so, might it be clearer to say `...includes host UID (%d)`?

Reply:
the same code is used both for UID and GID, that is why I used the generic ID :-)

I can pass down the information whether it is a UID or a GID and give a clearer error if you'd like

Reply:
added and pushed a new version

Reply:
Ah, thanks, that makes sense. Yes, please, I think users would appreciate a clearer message. Even after all this time, I still don't understand subuid/subgid, and I really would have no idea what a message like that would mean. Something like "the specified sub%cid mapping ... overlaps %cID %d" would make it unambiguous and easy to track down. Thank you!